python/doc/pyplots/P1LagrangeInterpolation.py

```diff
@@ -0,0 +1,30 @@
+import openturns as ot
+from openturns.viewer import View
+
+lowerBound = [0.0, 0.0]
+upperBound = [1.0, 1.0]
+interv = ot.Interval(lowerBound, upperBound)
+mesher = ot.IntervalMesher([6] * 2)
+mesh = mesher.build(interv, True)
+
+f = ot.SymbolicFunction(['x', 'y'], ['sin(pi_*x)*sin(pi_*y)'])
+values = f(mesh.getVertices())
+
+field = ot.Field(mesh, values)
+func_P1 = ot.P1LagrangeEvaluation(field)
+
```

Comment:
Maybe on this graph it could be nice to have a legend that explains the different type of lines

Reply:
ok. It is not easy... I put the information in the title... is it ok?